data-raw/simulating-DAG.R

```diff
@@ -28,18 +85,50 @@ simulated_data <- simulateSEM(graph, N = 2500) %>%
     setNames(paste0("metabolite_", 1:length(.)))
 
 devtools::use_data(simulated_data, overwrite = TRUE)
+>>>>>>> 6d0f48698be38b8c1d900db5c10e53eee6b09a3d
```

Comment:
You've got conflicts. I could ścreenshare and go through together today how to fix it.